common/prometheus-server/values.yaml

```diff
@@ -34,7 +34,7 @@ name:
 # names:
 
 # Defines how long data is stored. Format: `[0-9]+(ms|s|m|h|d|w|y)`
-retentionTime: 12h
+retentionTime: 25h
```

Comment:
This is volume issue safe?

Reply:
I don't expect big changes in volume size here. For Thanos this has no impact where volume size usually has the bigger impact.